src/resources/Platforn/PlatformInterfaces.ts

```diff
@@ -0,0 +1,32 @@
+import {UpdateStatusCategory} from '../Enums';
+
+export interface MaestroVersionOptions {
+    crawlingModuleVersion: string;
+}
+
+export interface CrawlingModule {
+    createdDate: 0;
```

Comment:
```suggestion
    createdDate: number;
```
Same for everywhere there is a `0`.